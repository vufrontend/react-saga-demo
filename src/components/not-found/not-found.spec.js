import React from 'react'
import NotFound from './not-found'

describe('NotFound', () => {
  it('Renders default', () => {
    const wrapper = shallow(<NotFound />)
    expect(wrapper).toMatchSnapshot()
  })
})
