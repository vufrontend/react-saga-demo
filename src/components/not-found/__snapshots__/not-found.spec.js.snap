// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Renders default 1`] = `
<h1>
  404
</h1>
`
